specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/runCommands.json

```diff
@@ -0,0 +1,467 @@
+          }
+        },
+        "label": {
+          "type": "string",
+          "description": "The VM run command label."
+        },
+        "description": {
+          "type": "string",
+          "description": "The VM run command description."
+        }
+      },
+      "required": [
+        "$schema",
+        "id",
+        "osType",
+        "label",
+        "description"
+      ],
+      "description": "Describes the properties of a Run Command metadata."
+    },
+    "RunCommandDocument": {
+      "allOf": [
+        {
+          "$ref": "#/definitions/RunCommandDocumentBase"
+        }
+      ],
+      "properties": {
+        "script": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "The script to be executed."
+        },
+        "parameters": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/RunCommandParameterDefinition"
+          },
+          "description": "The parameters used by the script."
+        }
+      },
+      "required": [
+        "script"
+      ],
+      "description": "Describes the properties of a Run Command."
+    },
+    "RunCommandListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/RunCommandDocumentBase"
+          },
+          "description": "The list of virtual machine run commands."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The uri to fetch the next page of run commands. Call ListNext() with this to fetch the next page of run commands."
+        }
+      },
+      "required": [
+        "value"
+      ],
+      "description": "The List Virtual Machine operation response."
+    },
+    "InstanceViewStatus": {
+      "properties": {
+        "code": {
+          "type": "string",
+          "description": "The status code."
+        },
+        "level": {
+          "type": "string",
+          "description": "The level code.",
+          "enum": [
+            "Info",
+            "Warning",
+            "Error"
+          ],
+          "x-ms-enum": {
+            "name": "StatusLevelTypes",
+            "modelAsString": false
+          }
+        },
+        "displayStatus": {
+          "type": "string",
+          "description": "The short localizable label for the status."
+        },
+        "message": {
+          "type": "string",
+          "description": "The detailed status message, including for alerts and error messages."
+        },
+        "time": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The time of the status."
+        }
+      },
+      "description": "Instance view status."
+    },
+    "RunCommandResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/InstanceViewStatus"
```

Comment:
>InstanceViewStatus [](start = 35, length = 18)

Why is this named InstanceViewStatus?

Reply:
These corresponds to a public class: InstanceViewStatus
https://docs.microsoft.com/en-us/dotnet/api/microsoft.azure.management.compute.models.instanceviewstatus?view=azure-dotnet